doc/flame/other/debug.md

```diff
@@ -35,3 +35,45 @@ commonly want to show the current FPS somewhere when the `FpsComponent` is used.
 
 
 [TextComponent]: ../rendering/text_rendering.md#textcomponent
+
+
+### ChildCounterComponent
+
+`ChildCounterComponent` is a component that can be added to a game and will every second render
+the number of children of type `T` from a component. So for example:
+
+```dart
+add(
+  ChildCounterComponent<SpriteAnimationComponent>(
+    target: world,
+  ),
+);
+```
+
+Will render the number of `SpriteAnimationComponent` that are children of the game `world`.
```

Comment:
It would be better to merge this with "So for example:" and put it above the code, since it sounds like it is continuing that sentence.
> So for example, the following will render the number of `SpriteAnimationComponent` that are children of the game `world`: